examples/grpc-bridge/README.md

```diff
@@ -0,0 +1,31 @@
+# envoy-grpc
+
+This is an example usage of the Envoy [gRPC bridge filter](https://lyft.github.io/envoy/docs/configuration/http_filters/grpc_http1_bridge_filter.html#config-http-filters-grpc-bridge). Included is a gRPC in memory Key/Value store with a Python HTTP client. The Python client makes HTTP/1 requests through the Envoy sidecar process which are upgraded into HTTP/2 gRPC requests. Response trailers are then buffered and sent back to the client as a HTTP/1 header payload.
```

Comment:
Can we potentially put this in the real docs alongside the other example? That would be cool.